tests/unit/conftest.py

```diff
@@ -26,6 +94,6 @@ def source_dir(tmpdir_factory):
 
 
 @pytest.fixture()
-def target_dir(tmpdir_factory):
+def target_dir(tmppath_factory: TempPathFactory):
     """ Return target directory path and clean up after tests """
-    return Path(tmpdir_factory.mktemp('target'))
+    return Path(tmppath_factory.mktemp('target'))
```

Comment:
Isn't `tmppath_factory.mktemp('target')` already returning an instance of `Path()`?